ui/views_test.py

```diff
@@ -252,6 +252,29 @@ def test_404_error_context_logged_out(self):
             assert response.context['authenticated'] is False
             assert response.context['name'] == ""
 
+    def test_500_error_context_logged_in(self):
```

Comment:
Can we add a test that causes a 500 error and verifies the page is shown, maybe by mocking something to raise an exception?